src-electron/zcl/zcl-loader-dotdot.js

```diff
@@ -947,12 +947,19 @@ async function processString(db, filePath, packageId, data) {
  * @param {*} dataType
  * @returns An Object
  */
-function prepareEnumsOrBitmaps(a, dataType) {
+function prepareEnumsOrBitmaps(a, dataTypeRef, dataType) {
+  if (a.type.toLowerCase().includes('uint')) {
+    let correctedType = dataType == dbEnum.zclType.enum ? 'enum' : 'map'
+    a.type = a.type.toLowerCase().replace('uint', correctedType)
+    env.logWarning(
+      `Warning: ${correctedType} ${a.name} is declared as uint in Unify XMLs. Replacing with ${correctedType} -${a.type}`
```

Comment:
`Warning: ${a.name} is declared incorrectly as ${a.type} in XML. Replace ${a.type} with ${correctedType}`